### ISSUE TITLE
Example for AccessURL headers dpesn't match schema

### ISSUE THREAD

**Issue by jessebrennan** [ga4gh/data-repository-service-schemas#307]
It looks like this is [specified here](https://github.com/ga4gh/data-repository-service-schemas/blob/7b8e94d01b72f4fbaa96fdddfb14c8e71420ce50/openapi/data_repository_service.swagger.yaml#L320) in the OpenAPI spec.

The type is listed as an Array of strings, but the example is on object with keys as header keys and values as header values. The example definitely makes more sense, and my guess is that the schema should be changed.

**Comment by jfuerth** [comment]
Just noticed this today! FWIW, we have already implemented clients and servers that model AccessURL headers according to the example (as an object, not an array). We will stick with this for now.

**Comment by briandoconnor** [comment]
Yep, I see the inconsistency in the spec now.  I agree, since folks are implementing based on the example and it's clear how to go from a key-value hash to [HTTP headers](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields) we should go with the object instead of the array.  Time to make a PR and discuss on the Cloud Work Stream call

**Comment by u8sand** [comment]
I also got confused with this; notably the ga4gh-starter-kit is also treating it as an object.

https://github.com/ga4gh/ga4gh-starter-kit-drs/blob/47a13d53c6e505964c1f9e4157eb491fd1fab296/src/main/java/org/ga4gh/starterkit/drs/model/AccessURL.java#L21

I second making it an object..

**Comment by jb-adams** [comment]
+1 for object instead of array of strings, as @u8sand pointed out that's what we've done in the starter kit
